### DISCUSSION TITLE
Horilla license

### DISCUSSION THREAD

**Discussion by leonliuaptean** [horilla-opensource/horilla#388] [Q&A]
Hi All,

quick question on Horilla license.  Is it LGPL-2.1 or AGPL?  I thought it is LGPL-2.1 license, but in the read me file it uses AGPL license picture.
![image](https://github.com/user-attachments/assets/b40ed05c-1828-4367-8645-8cd05b9c374e)

**Comment by horilla-opensource** [comment]
Hi @leonliuaptean ,

The license is LGPL-2.1 itself. It was a mistake from our end in README.md file. We have updated it.

Thanks for pointing it out and we appreciate your interest in Horilla.

Do let us know if you have any queries or concerns.

With Regards,
**Team Horilla**
